ansible/files/hetzner_server_nftables.conf

```diff
@@ -0,0 +1,32 @@
+#!/usr/sbin/nft -f
```

Comment:
general question: why did you go with this method, rather than ufw rules?

Reply:
another point here: how do we (temporarily) turn on masquerading for this setup?